### DISCUSSION TITLE
Cant seem to get this one working, anyone else?

### DISCUSSION THREAD

**Discussion by Armagedon622** [Arnuh/RuneLiteHijack#6] [General]
As the title says.

**Comment by Arnuh** [comment]
Provide logs and create an issue. Otherwise I can't do much.

**Reply by Armagedon622** [reply]
Could you maybe PM me your discord or something, i dont really know how to send logs etc, ive done everything exactly as instructed, and when i run the program nothing happens.
edit: Maybe the reason is that there is no runelitehijack.jar file?
![image](https://user-images.githubusercontent.com/126707765/222761310-472232ac-dd40-4d40-ae3b-5f6e0ccb7425.png)

**Reply by Arnuh** [reply]
https://github.com/Arnuh/RuneLiteHijack/releases/tag/latest

**Reply by Armagedon622** [reply]
> https://github.com/Arnuh/RuneLiteHijack/releases/tag/latest

Alright i got the hijack plugin hub, but cant figure out how to add custom plugins to it tho.
Tried adding plugins to everywhere yet they aint appearing xD.
![image](https://user-images.githubusercontent.com/126707765/222883839-a4308aad-4a64-4059-892e-0009362a5017.png)

**Reply by Arnuh** [reply]
This project only adds the ability to create custom "Plugin Hubs" which https://github.com/Arnuh/RuneLiteHijack/issues/3 somewhat describes. Anything else is a feature request and should have an issue created for such request.
